coc-incident-resolution-procedures-proposed.md

```diff
@@ -162,7 +167,7 @@ If any party does not fulfill their obligations related to resolution of the inc
 
 ### Communicating the Results
 
-An incident is considered “resolved” when the CoC Committee has completed its investigation and either (a) determined what remediation actions are needed to resolve an incident (including determining that the involved parties’ agreed-upon resolution is adequate) (b) or determined that the CoC was not violated and no remediation is needed. When the incident is resolved, a member of the CoC Committee will inform the person who submitted the report. The CoC Committee will determine how much information to share with the reporter regarding the committee’s findings and what remediation steps were taken, if any, taking into consideration the privacy, confidentiality, and potential impact to the individuals involved in the incident. The CoC Committee will also determine what information, if any, is necessary to share publicly or with community and project leaders. Also see “Information Sharing” below.
+An incident is considered “resolved” when the CoC Committee has completed its investigation and either (a) determined what remediation actions are needed to resolve an incident (including determining that the involved parties’ agreed-upon resolution is adequate) (b) or determined that the CoC was not violated and no remediation is needed. When the incident is resolved, a member of the CoC Committee will inform the person who submitted the report. The CoC Committee will determine how much information to share with the reporter regarding the committee’s findings and what remediation steps were taken, if any, taking into consideration the privacy, confidentiality, and potential impact to the individuals involved in the incident. After resolution of the incident, the CoC Committee shall determine in its discretion whether, how, and when to notify the accused person. The CoC Committee shall The CoC Committee will also determine what information, if any, is necessary to share publicly or with community and project leaders. Any communication regarding the results of the investaigtion shall comply with the [Confidentiality Policy](#confidentiality-policy). Also see [Information Sharing](#information-sharing) below.
```

Comment:
"The CoC Committee shall The CoC Committee will also determine what information"

There's a stutter in this text

Reply:
My personal opinion: If an accused person was notified of an incident that the CoCC is investigating, I believe they should be notified at a minimum that the incident was closed and what, if any action, is being taken that would impact them. I feel the current wording is a bit to permissive as it basically says the CoCC can do anything they want within their discression.

Reply:
Thanks @cblecker for your comments and catching & fixing the typo. My PR to modify the language was made in response to a comment from @tpepper (https://github.com/cncf/wg-coc/discussions/15#discussioncomment-4382000) that the notification of outcome should mirror notification that an investigation has been opened.

@tpepper, feel free to chime in if I misunderstood what your comment was trying to achieve.

My understanding of the current language is that the committee has a lot of discretion because sometimes a committee will decide not to notify the accused during the investigation to prevent witness tampering or protect reporters (or the committee itself from retaliation; if the outcome of the investigation is that there was no violation and therefore there are no consequences for the accused person, the committee might decide for similar reasons not to notify the accused person that there ever was a complaint filed. Obviously, if there are going to be consequences that impact an accused person, they would need to be notified.

As an example, years ago, I helped a CoC committee with an investigation against someone who was extremely unpredictable, threatening, and abusive. This person left the community on their own, so the committee decided not to notify them of the CoC finding unless, but would do so if this person tried to re-join the community at a later date. 
 It was determined that any attempt to communicate with this person (by email, phone, or otherwise) could expose committee members to a risk of abuse or even violence from this person, so it would be better not to further provoke them given that they had left the community on their own.
 
 These corner cases come up from time to time, and sometimes it's helpful for a committee to have flexibility.

Reply:
I can understand the desire for flexibility on *if* you notify an accused person. In addition to the reasons you specify, I would also want to allow the CoCC the flexibility to say "this report is has so little merit to it that we feel we can safely close it without speaking to the accused person".

The case that myself and @tpepper are trying to cover is if/when an accused person is contacted by the CoCC in order to investigate an incident, I think there needs to be closure on the other side. If the CoCC decides to take no action, even just a "hey, we are closing this file" can, again, help all sides to heal from something. Having an open ended, unresolved investigation hanging over an individuals head isn't fair to them either.

Reply:
@cblecker I agree that if someone is notified that a complaint has been filed, that they need to be notified of the final outcome once the incident has been resolved